### DISCUSSION TITLE
json vs xml metadata main difference

### DISCUSSION THREAD

**Discussion by Danny-dK** [UtrechtUniversity/yoda#207] [Q&A] [answered]
I received a question on json vs xml format for metadata. Within Yoda, there was a switch from xml to json. What is the main difference or benefit to use json formats? Is there for example a switch to json formats for metadata happening world-wide, or was it more practicality within Yoda. When dealing with metadata outside of Yoda, would there be a preference to use one format over the other?

**Comment by lwesterhof** [comment, marked as answer]
There are several benefits for using JSON for our metadata:
- JSON is somewhat more human readable then XML
- DataCite introduced DataCite JSON: https://blog.datacite.org/introducing-datacite-json/
- All the libraries we use support JSON, switching from XML to JSON removed the need to transform between the two formats
- We can use JSON schema (https://json-schema.org/)
